src/resp_parser.c

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright (c) 2009-2021, Redis Labs Ltd.
```

Comment:
On another thread, https://github.com/redis/redis/issues/9236, I talked with some OSS lawyers and they suggested just putting `Copyright Redis Contributors.` for new files. The copyright years are not required for the most part. Feel free to to include Redis Labs Ltd. if you feel it's important.

Reply:
We're also checking this issue, let's stick with what we've done so far until we have an agreement on anything new.